podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Controllers/GameController.cs

```diff
@@ -8,33 +8,29 @@ public class GameController : MonoBehaviour
     public Modal modal;
     public string userName;
     public bool isModalPositive;
-
     void Start()
     {
         
     }
-
     void Update()
     {
         modalControl();
-        
     }
-
+    bool IsWaitingForAnswert = true;
```

Comment:
Let's move this variable before all methods. I propose to change name to isWaitingForAnswer